include/moqt/server.h

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright (C) 2024
+ *  Cisco Systems, Inc.
+ *  All Rights Reserved
+ */
+
+#pragma once
+
+#include <moqt/config.h>
+#include <moqt/core/messages.h>
+#include <moqt/core/transport.h>
+
+namespace moq::transport {
+    using namespace qtransport;
+
+    /**
+     * @brief MoQT Server
+     *
+     * @details MoQT Server is the handler of the MoQT QUIC listening socket
+     */
+    class Server : public Transport
```

Comment:
same comment as is Server a Transport or Does Server has a Transport handle

Reply:
`moq::transport::Transport` is common and used by both `moq::transport::Client` and `moq::transport::Server`. We could add `Handler` as a suffix to both `Client` and `Server` but that didn't seem to be needed as it gets weird with everything having handler in the name.

Reply:
I see inheritance here, i was suggesting for composition